CMakeLists.txt

```diff
@@ -272,6 +272,8 @@ if(ENABLE_MAD)
     add_definitions(-DENABLE_MAD)
     include_directories(${MAD_INCLUDE_DIR})
     list(APPEND XOREOS_LIBRARIES ${MAD_LIBRARY})
+  else()
+	set(ENABLE_MAD OFF)
```

Comment:
We're using spaces instead of tabs for indenting in the CMakeLists.txt, so please change the tabs you added in this commit with spaces.

(Yes, that's a break with the usual xoreos style. I simply didn't catch that when I accepted the PR that adds the CMake files. Adding a commit on top that changes all indentation to tabs there would be a bit messy.)

Also, please remove the period at the end of the commit message for this commit. The first line in a commit message shouldn't have a period in the xoreos repositories.